cmd/libsnap-confine-private/snap-test.c

```diff
@@ -317,8 +319,8 @@ static void test_sc_instance_name_validate(void)
 	sc_error_free(err);
 
 	const char *valid_names[] = {
-		"a", "aa", "aaa", "aaaa",
-		"a_a", "aa_1", "a_123", "a_0123456789",
```

Comment:
Interesting that short snap name with long instance key is still too short. Useful test.